apps/tailwind-components/pages/input/Listbox.story.vue

```diff
@@ -0,0 +1,146 @@
+    </p>
+  </article>
+  <form class="mb-6 bg-white rounded p-4">
+    <legend class="mb-2 text-heading-lg">
+      Configure the listbox component
+    </legend>
+    <div class="flex flex-row flex-wrap justify-center items-start gap-4">
+      <div class="p-2 grow">
+        <InputLabel for="listbox-data-type" class="pl-0">
+          <span>Select the input data type</span>
+        </InputLabel>
+        <InputRadioGroup
+          id="listbox-data-type"
+          v-model="listboxDataType"
+          :radio-options="[
+            { value: 'string', label: 'String array' },
+            { value: 'array', label: 'Array of options' },
+          ]"
+        />
+      </div>
+      <div class="p-2 grow">
+        <InputLabel for="listbox-component-state" class="pl-0">
+          Change the component state
+        </InputLabel>
+        <InputRadioGroup
+          id="listbox-component-state"
+          v-model="listboxState"
+          :radioOptions="[
+            {
+              value: 'disabled',
+              label: 'Disabled: disable interactivity with the component',
+            },
+            {
+              value: 'error',
+              label: 'Error: simulate an instance where an error occurred',
+            },
+          ]"
+          :showClearButton="true"
+        />
+      </div>
+    </div>
+    <div class="bg-white p-2 grow">
+      <InputLabel for="listbox-placeholder">
+        Change the default placeholder text
+      </InputLabel>
+      <InputString id="listbox-placeholder" v-model="listboxPlaceholder" />
+    </div>
+  </form>
+  <div class="mb-6 bg-white rounded p-6">
+    <h3 class="text-heading-lg mb-2">Listbox example</h3>
+    <InputLabel
+      id="listbox-input-label"
+      for="listbox-input"
+      class="block mb-3 pl-0"
+    >
+      <span>Select a group assignment</span>
+    </InputLabel>
+    <InputListbox
+      id="listbox-input"
+      labelId="listbox-input-label"
+      :options="listboxData"
+      v-model="listboxSelection"
+      :hasError="listboxState === 'error'"
+      :disabled="listboxState === 'disabled'"
+      :placeholder="listboxPlaceholder"
+    />
+    <output class="block w-full mt-6 bg-gray-100 py-3 px-2 pl-6">
+      <code
+        >Output {{ listboxDataType === "true" ? "Value" : "Object" }}:
+        {{ listboxSelection }}</code
+      >
+    </output>
+  </div>
+  <div class="mb-2 bg-white rounded p-6">
+    <h3 class="text-heading-lg mb-2">Input data structure</h3>
+    <p>Based on the selection above, the input data is shown below.</p>
+    <output
+      class="block w-full mt-6 bg-gray-100 py-3 px-2 pl-6 h-60 overflow-y-scroll shadow-inner"
+    >
+      <pre class="indent-[-5em]">
+        {{ listboxData }}
+      </pre>
+    </output>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import { ref, computed } from "vue";
+import type { IListboxValueArray, IListboxOption } from "../../types/listbox";
+
+// as ListboxOption
```

Comment:
what do you mean?

Reply:
Ah, forgot to delete this. I believe this is from an earlier example. Will remove it.